src/main/java/meteordevelopment/meteorclient/utils/player/SlotUtils.java

```diff
@@ -36,26 +37,26 @@ public static int indexToId(int i) {
         ScreenHandler handler = mc.player.currentScreenHandler;
 
         if (handler instanceof PlayerScreenHandler) return survivalInventory(i);
```

Comment:
This can break if you just yeet all the `else if`
Same comment as above, unneeded change